### ISSUE TITLE
How do I mount a folder as a CD-ROM for Windows 98?

### ISSUE THREAD

**Issue by BridgeHeadland** [joncampbell123/dosbox-x#4082]
### Question

imgmount c -t hdd xxxxxxxx.img
imgmount d -t hdd yyyyyyyy.img
imgmount e -t hdd zzzzzzzz.img
boot -L c:

This is my auto executable command I use to run Windows 98 in DOSBox. In my case, I can also open the same CD-ROM drive as in my main operating system (Windows 10), but I want to remount one of my image file to a virtual CD-ROM drive, to avoid having to use CDs all the time. I know I have to rewrite the command line imgmount 'YYYYYYYY' or 'ZZZZZZZZ' so that the image file behaves like a CD-ROM, I just don't know how.

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by Torinde** [comment]
Per the [wiki](http://dosbox-x.com/wiki/Guide:Managing-image-files-in-DOSBox‐X): you can try with `imgmount D: yyyyyyyy.img -t cdrom`

**Comment by rderooy** [comment]
Even the `-t cdrom` option is not needed, if the file extension is a recognised extension like `.iso` or `.cue`. And for the drive to mount it on, you can even get away with just the letter, without `:` suffix.
e.g.,
```
imgmount d game.iso
```

**Comment by BridgeHeadland** [comment]
@Torinde Sorry, but when I say image files, I also mean bin/cue and iso files. I don't know which category to put IMG, BUN/CUE and ISO in, that's why I call them all image files. :)

To me it doesn't matter if it's a folder, or an IMG, BIN/CUE and ISO, which behaves like a virtual CD-ROM drive, only the Windows 98 operating system thinks that a specific CD is in the actual computer, without the CD actually being inside.
I tried the command lines, both with and without "-t cdrom" none of them worked, and I tried mounting both an IMG file and an ISO file.

**Comment by rderooy** [comment]
Did you check the wiki? In particular the known limiations part?
https://dosbox-x.com/wiki/Guide%3AInstalling-Windows-98#_outstanding_issues

**Comment by BridgeHeadland** [comment]
@rderooy No, I didn't, but I see now that it's impossible for now. It would be much easier if one could run Windows 95 and later, just not as guest OSes.

The reason I want to mount an IMG, BIN/CUE or an ISO, as a CD-ROM is because I believe that the software will have a lower risk of crashing, if Windows 98SE thinks there is a real CD in the computer, which is a virtual computer, instead of the files from the CDs being in an otherwise empty drive D, for example.
To be honest, I use No CD fix files for the game, which is the likely reason why the game crashes during the minigames and activities, in the game, which use load times, probably because there is supposed to be a CD in the computer.
I am uploading the No CD fix files here, . I'm hoping someone here can figure out what could be causing the game to crash. The reason for using the No CD fix is ​​because the game won't start without them, for some reason. You can, for example, open the files with a HEX editor.
[FGP_GOLD No CD fix.zip](https://github.com/joncampbell123/dosbox-x/files/11009254/FGP_GOLD.No.CD.fix.zip)

**Comment by rderooy** [comment]
@BridgeHeadland I fail to see what your rant has to do with my comment. You just need to add a flag to the `imgmount` command as documented there.

**Comment by BridgeHeadland** [comment]
@rderooy I must have read it wrong, sorry.

**Comment by Torinde** [comment]
@rderooy, the Wiki guides for 98 and ME mention 95 in their "Known limitations" sections:
![image](https://user-images.githubusercontent.com/97228894/226159717-d78c909a-1c20-4fcb-96f8-3c1fedf0aef0.png)

Should those 95 references be changed to 98/ME?

**Comment by rderooy** [comment]
Yes, I had already updated it in the draft version for the next release

**Comment by BridgeHeadland** [comment]
Yesterday I added No CD fix files to the gold edition of a game with a Norwegian name (PC-spillet om Flåklypa Grand Prix). I just wanted to remind you.
If those who agree to fix the files need to compare them with the original files, I will upload them here and now.
[FGP_GOLD from CDs.zip](https://github.com/joncampbell123/dosbox-x/files/11011968/FGP_GOLD.from.CDs.zip)

**Comment by BridgeHeadland** [comment]
Yesterday I uploaded the No CD fix files for the gold edition of the game with the Norwegian name PC-spillet om Flåklypa Grand Prix, which I hoped some experts could examine with a HEX editor, because the game crashed on the activities and minigames where it is used a lot loading time, unless there is no sound in Windows 98. I think I should remind you of that.
I am now uploading the original executable so you can compare it with the No CD fix executable.
[FGP.ZIP](https://github.com/joncampbell123/dosbox-x/files/11012212/FGP.ZIP)

**Comment by github-fanuser** [comment]
I'm not sure what you are wanting to accomplish, There is a program called Virtual Clone Disk. I believe v5.4.5 supports Win98. Also, There is a program called ImDisk, I believe there is an older version that will run on Windows 98. Hope this helps. Good luck!
